cmd/root.go

```diff
@@ -70,9 +72,37 @@ Once it has received the token in the response from Vault, it will encrypt and
 store this token on S3 from where it can be used for authentication by entities that
 need to read from or write to the Vault instance.`,
 	Run: func(cmd *cobra.Command, args []string) {
+		vaultAddr := os.Getenv("VAULT_ADDR")
+		if vaultAddr == "" {
+			vaultAddr = "http://127.0.0.1:8200"
+		}
+
+		httpClient = http.Client{
+			Transport: &http.Transport{
+				TLSClientConfig: &tls.Config{
+					InsecureSkipVerify: false,
```

Comment:
Should we rather be encrypting traffic to/from vault?

Reply:
I figured this is internal traffic and is ok? 😅

Reply:
By the way are you referring to putting self-signed certs etc?

Reply:
> I figured this is internal traffic and is ok? 😅

I don't think its okay, we should always have TLS enabled for important components such as Vault. 

@hadrianvalentine Was this added perhaps because you were testing with an insecurely configured vault that lacked TLS?

Reply:
> I don't think its okay, we should always have TLS enabled for important components such as Vault.

Noted. That said, I realise now that this is actually `false` (he initially had it as `true`, see https://github.com/Intellection/vault-init/pull/1#discussion_r332455583. Is the issue that *we should be using https internally* ... and not this specific line ... because this is set not to ignore certificate verification, which I'm guessing is what we want? 🤔

Reply:
> That said, I realise now that this is actually `false` (he initially had it as `true`, see [#1 (comment)](https://github.com/Intellection/vault-init/pull/1#discussion_r332455583).

Ah right, I'm mistaken then. Usually you'd only set this if you're turning it off hence I jumped to an assumption.

We should probably delete this code then?

Reply:
Yeah, this can be deleted. Removing...